sql/core/benchmarks/DataSourceReadBenchmark-results.txt

```diff
@@ -1,431 +1,431 @@
-SQL CSV                                           11407          11426          26          1.4         725.3       1.0X
-SQL Json                                           9654           9660           7          1.6         613.8       1.2X
-SQL Parquet Vectorized: DataPageV1                   91            100           7        172.0           5.8     124.7X
-SQL Parquet Vectorized: DataPageV2                   88             97           8        178.6           5.6     129.5X
-SQL Parquet MR: DataPageV1                         1913           1916           5          8.2         121.6       6.0X
-SQL Parquet MR: DataPageV2                         1868           1871           5          8.4         118.7       6.1X
-SQL ORC Vectorized                                  115            124           6        136.7           7.3      99.1X
-SQL ORC MR                                         1519           1523           6         10.4          96.6       7.5X
-
-OpenJDK 64-Bit Server VM 17.0.11+9-LTS on Linux 6.5.0-1018-azure
+SQL CSV                                           11538          11589          73          1.4         733.5       1.0X
+SQL Json                                           9586           9596          14          1.6         609.5       1.2X
+SQL Parquet Vectorized: DataPageV1                  109            116           6        144.8           6.9     106.2X
+SQL Parquet Vectorized: DataPageV2                  110            118           8        142.6           7.0     104.6X
+SQL Parquet MR: DataPageV1                         1901           1953          74          8.3         120.9       6.1X
+SQL Parquet MR: DataPageV2                         1817           1832          22          8.7         115.5       6.4X
+SQL ORC Vectorized                                  118            126           7        133.6           7.5      98.0X
+SQL ORC MR                                         1505           1535          43         10.5          95.7       7.7X
+
+OpenJDK 64-Bit Server VM 17.0.11+9-LTS on Linux 6.5.0-1022-azure
 AMD EPYC 7763 64-Core Processor
 Parquet Reader Single TINYINT Column Scan:   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ---------------------------------------------------------------------------------------------------------------------------
-ParquetReader Vectorized: DataPageV1                    69             71           1        227.3           4.4       1.0X
-ParquetReader Vectorized: DataPageV2                    69             71           2        228.4           4.4       1.0X
-ParquetReader Vectorized -> Row: DataPageV1             47             48           1        332.4           3.0       1.5X
-ParquetReader Vectorized -> Row: DataPageV2             47             48           1        334.0           3.0       1.5X
+ParquetReader Vectorized: DataPageV1                    93             94           1        169.9           5.9       1.0X
```

Comment:
From the result , it appears that there is a slight decrease in throughput in the `TINYINT` scenario.
cc @dongjoon-hyun @sunchao

Reply:
I guess this is probably due to other factors. It's fine as long as the relative stay the same?